README.md

```diff
@@ -60,6 +60,7 @@ git clone https://github.com/Graph-Learning-Benchmarks/gli.git
 cd gli
 pip install -e .
 ```
+MacOS users may need to install `wget` in terminal to avoid `FileNotFoundError: [Errno 2] No such file or directory: 'wget'`. Run `brew install wget` to solve this problem (you should have [homebrew](https://brew.sh/) installed first)
```

Comment:
Let's specify that `wget` is required to download GLI datasets (whatever the system is). Users should be able to find download instructions easily according to their os.